examples/solvers/functional_basic_example.py

```diff
@@ -4,11 +4,7 @@
 information on functionals, see `the ODL functional guide
 <http://odlgroup.github.io/odl/guide/in_depth/functional_guide.html>`_
 """
-
-# Imports for common Python 2/3 codebase
-from __future__ import print_function, division, absolute_import
-from future import standard_library
-standard_library.install_aliases()
+from __future__ import print_function, division
```

Comment:
Well print with parenthesis works as well in py2 as py3, what the function does is allow it to be used as a first class object, e.g. pass it as an argument to a function. But you don't do that here.